common/src/main/java/com/wynntils/models/gear/type/SetInstance.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.gear.type;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.models.stats.type.StatType;
+import java.util.Map;
+
+// wynncraftCount can eventually be removed when Wynncraft fixes their double ring bug...
```

Comment:
This is actually backwards. True count can be removed when Wynn fixes their issue, probably along with the non-automatically updated set data, since it won't be accurate. We should rely on in-game parsed (true) data, unless the API has set info data.